frontend/src/pages/org/settings/components/billing.ts

```diff
@@ -0,0 +1,268 @@
+            html`
+              <p class="mb-3">
+                ${msg(
+                  "Subscription status, features, and add-ons, if applicable.",
+                )}
+              </p>
+              ${when(
+                this.org,
+                (org) => html`
+                  <p class="leading-normal">
+                    ${org.subscription
+                      ? html`${msg(
+                          str`You can view plan details, update payment methods, and update billing information by clicking “${this.portalUrlLabel}”.`,
+                        )}
+                        ${this.salesEmail
+                          ? msg(
+                              html`To upgrade to Pro, contact us at
+                                <a
+                                  class=${linkClassList}
+                                  href=${`mailto:${this.salesEmail}?subject=${msg(str`Upgrade Starter plan (${this.org?.name})`)}`}
+                                  rel="noopener noreferrer nofollow"
+                                  >${this.salesEmail}</a
+                                >.`,
+                            )
+                          : nothing}`
+                      : this.salesEmail
+                        ? msg(
+                            str`Contact us at ${this.salesEmail} to make changes to your plan.`,
+                          )
+                        : msg(
+                            str`Contact your Browsertrix host administrator to make changes to your plan.`,
+                          )}
+                  </p>
+                `,
+              )}
+            `,
+          ],
+        ])}
+      </div>
+    `;
+  }
+
+  private readonly renderSubscriptionDetails = (
+    subscription: OrgData["subscription"],
+  ) => {
+    let tierLabel;
+    let statusLabel;
+
+    if (subscription) {
+      tierLabel = html`
+        <sl-icon class="text-neutral-500" name="nut"></sl-icon>
+        ${msg("Starter")}
+      `;
+
+      switch (subscription.status) {
+        case SubscriptionStatus.Active: {
+          if (subscription.futureCancelDate) {
+            statusLabel = html`
+              <span class="text-warning-600"
+                >${msg(
+                  html`Canceling on
+                    <sl-format-date
+                      lang=${getLocale()}
+                      class="truncate"
+                      date=${subscription.futureCancelDate}
+                      month="2-digit"
+                      day="2-digit"
+                      year="2-digit"
+                    >
+                    </sl-format-date>`,
+                )}</span
+              >
+            `;
+          } else {
+            statusLabel = html`
+              <span class="text-success-700">${msg("Active")}</span>
+            `;
+          }
+          break;
+        }
+        case SubscriptionStatus.PausedPaymentFailed: {
+          statusLabel = html`
+            <span class="text-warning-600">
```

Comment:
This should probably be `text-danger`?  Would place "payment failed" in the "severe" error category?